glusterfs/meta/heka.yml

```diff
@@ -0,0 +1,53 @@
+log_collector:
+  decoder:
+    glusterfs:
+      engine: sandbox
+      module_file: /usr/share/lma_collector/decoders/glusterfs.lua
+      module_dir: /usr/share/lma_collector/common;/usr/share/heka/lua_modules
+      adjust_timezone: true
+  input:
+{% if pillar.glusterfs.server is defined %}
+{% from "glusterfs/map.jinja" import server with context %}
+{%- if server.enabled %}
```

Comment:
You use `cilent.get('enabled', False)` below, don't you want to use the same pattern here?